src/iotjs_binding_helper.c

```diff
@@ -171,5 +171,12 @@ int iotjs_process_exitcode() {
 
 void iotjs_set_process_exitcode(int code) {
   const jerry_value_t process = iotjs_module_get("process");
-  iotjs_jval_set_property_number(process, IOTJS_MAGIC_STRING_EXITCODE, code);
+  jerry_value_t jstring =
+      jerry_create_string((jerry_char_t*)IOTJS_MAGIC_STRING_EXITCODE);
+  jerry_value_t jcode = jerry_create_number(code);
+  jerry_value_t ret_val = jerry_set_property(process, jstring, jcode);
```

Comment:
A bit shorter way: `jerry_release_value(jerry_set_property(process, jstring, jcode));`

Reply:
Yes, it's definitely shorter, but I don't know if it looks better.
However you are right, since the return value would be freed anyway.